.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,39 @@
+name: Unittests
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: install dev deps
+      run: pip install -e ".[dev]"
+    - name: develop
+      run: python setup.py develop
```

Comment:
This isn't needed:
`evol 0.4.1 is already the active version in easy-install.pth`

Reply:
Removed it.